src/source/campaign-overview.rst

```diff
@@ -0,0 +1,86 @@
+.. _campaign_overview:
+
+Overview
+========
+
+Campaigns allow you, the seller, to group and manage your ads together, having shared budgets, targeting rules, and other settings. You can create separate campaigns to run ads in different locations or using different budgets.
+
+Campaigns are a simpler way to organize and manage your products, services, and ads in general.
+They are the standard for organising your ads, and they will serve as the primary container to get your ads online.
```

Comment:
this section is written as talking to a seller, whereas budgets / linked ads are written more objectively. Do we care about such a thing and do we make it more consistent, or will we leave it as is?

Reply:
good point, let me revise it

Reply:
btw we have that sort of inconsistency already in the old specs (look at the Advertisements part vs the rest)